SteamKit2/SteamKit2/Types/JobID.cs

```diff
@@ -6,10 +6,12 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Diagnostics;
```

Comment:
[where] is this needed?

Reply:
its not. gonna need to do a usings cleanup pass, because a lot of files have unused ones.